lib/tmp.js

```diff
@@ -289,7 +292,7 @@ function fileSync(options) {
   var fd = fs.openSync(name, CREATE_FLAGS, opts.mode || FILE_MODE);
   /* istanbul ignore else */
   if (opts.discardDescriptor) {
-    fs.closeSync(fd); 
+    fs.closeSync(fd);
```

Comment:
This is an editor-generated change, let me know if you want me to remove it from this patch

Reply:
No, this is fine, thank you!